concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2994,35 +3004,14 @@ jobs:
       - get: smtp-credential-rotation-timer
         trigger: true
     - task: forget-ses-smtp-access-key
-      config:
-        platform: linux
-        image_resource:
-          type: docker-image
-          source:
-            repository: governmentpaas/terraform
-            tag: 9cad30b5d5889a0b72173f39701d1620e24df82c
-        inputs:
-          - name: cf-tfstate
-        outputs:
-          - name: updated-tfstate
-        params:
-          AWS_DEFAULT_REGION: ((aws_region))
-        run:
-          path: sh
-          args:
-            - -e
-            - -c
-            - |
-              cp cf-tfstate/cf.tfstate updated-tfstate/cf.tfstate
-              terraform state rm \
-                -state=updated-tfstate/cf.tfstate \
-                aws_iam_access_key.ses_smtp
-              echo "ses smtp access key no longer managed by terraform."
-              echo "next successful deployment run will revoke the expired keys"
+      file: paas-cf/concourse/tasks/forget-ses-smtp-access-key.yml
```

Comment:
You need to pull the resource `paas-cf`, add a `get: pass-cf` before the task.

Reply:
![image](https://user-images.githubusercontent.com/280032/35151469-5988f24c-fd16-11e7-92e4-146ab32b9ab4.png)